playbooks/generic-tempest-output-to-junit.yml

```diff
@@ -0,0 +1,83 @@
+---
+- hosts: soc-deployer
+  gather_facts: yes
+  tasks:
+    - name: Run container to attach tempest pvc
+      shell: |
+        set pipefail -o
+        cat <<EOF | kubectl -n openstack apply -f -
+        apiVersion: v1
+        kind: Pod
+        metadata:
+          name: tempest-data
+        spec:
+          containers:
+          - name: tempest-data
```

Comment:
If you move to nginx image, you could expose the /data. You can then curl the data, though I wouldn't expect this to be necessary. You could keep the current loop.

Reply:
using suse containers all the way babyyy.

Reply:
It's fine for me.